### PR TITLE
Add the final modification

### PR DESCRIPTION
**Pull request by Chengyuan-artist**
- add try-catch

- modify the xml

- delete the unnecessary code

### CODE REVIEW

**Review by Chengyuan-artist — COMMENTED** [review]
nienie

**Comment by Chengyuan-artist** [comment]
a nuo

File: app/src/main/java/tcc/bbshust/ui/postdetail/PostDetailViewModel.kt [hunk 1]
```diff
@@ -69,23 +79,30 @@ class PostDetailViewModel(
     }
 
     fun addReply(content: String) {
-        uiScope.launch {
```
Comment (xzcxzcyy) [inline app/src/main/java/tcc/bbshust/ui/postdetail/PostDetailViewModel.kt:72]: should catch inside a coroutine

**Review by xzcxzcyy — CHANGES_REQUESTED** [review]

**Review by xzcxzcyy — APPROVED** [review]
wa ka li masta

File: app/src/main/java/tcc/bbshust/ui/postdetail/PostDetailViewModel.kt (hunk 1 above)
Comment (Chengyuan-artist) [reply app/src/main/java/tcc/bbshust/ui/postdetail/PostDetailViewModel.kt:72]: copy that

**Pull request merged**